src/pages/sorry.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import $box from "@/styles/Box.module.css";
+import $typo from "@/styles/Typography.module.css";
+
+const SorryPage: React.FC = () => {
```

Comment:
If we aren't going to use this element in this PR, let's remove it and add it to a new branch split from this one.